src/pages/postgraphile/make-extend-schema-plugin.md

```diff
@@ -535,6 +535,62 @@ const MyRegisterUserMutationPlugin = makeExtendSchemaPlugin(build => {
 Note that the `@pgField` directive here is necessary for PostGraphile to "look
 ahead" and determine what to request from the database.
 
+#### Working with arrays via `json_array_elements`
+
+Here's an example of working with a join table, and bulk inserting multiple records from a graphQL List.
+
+```js
+...
+
+typeDefs: gql`
+  input updatePersonsThingsInput {
+    personId: UUID!,
+    thingIds: [UUID!]!
+  }
+  extend type Mutation {
+    updatePersonsThings(input: updatePersonsThingsInput!): PersonsThingsConnection
+  }
+`,
+resolvers: {
+  Mutation: {
+    updatePersonsThings: async (_query, { input: { personId, thingIds } }, { pgClient }, resolveInfo) => {
+      await pgClient.query("SAVEPOINT graphql_mutation");
+      try {
+        // Ensure proper formatting. This may not be necessary if not modifying the input
+        const elements = JSON.stringify(thingIds.map(thingId => ({ thingId, personId })));
+        
+        // Bulk insert
+        await pgClient.query(`
```

Comment:
```suggestion
        const { rows } = await pgClient.query<{ id: number }>(`
```

Reply:
`<{ id: number }>` What's this?

Reply:
Yeah, if the example is in JS then you should drop it, but for TS it helps us to know the type we're expecting postgres to return.